coverage-config

```diff
@@ -0,0 +1,6 @@
+# .coveragerc to control coverage.py
+[run]
+omit=*/migrations/*,*/mocks/*,*/tests*,*/management/*,*/wsgi.py,*/admin.py,manage.py,*/settings/*,*/apps.py,*/__init__.py
```

Comment:
Please add our integration tests (`api-test.py` and `api-integration-test.py`) here. They're at 0% coverage and I think they'll stay at that because, after all, we won't be writing tests for tests :)

Also, is it possible to wrap this into multiple lines? Just like with imports, it would make future diffs easier to read.

Reply:
:+1: